go.mod

```diff
@@ -3,49 +3,65 @@ module github.com/vmware-tanzu/velero-plugin-for-aws
 go 1.18
 
 require (
-	github.com/aws/aws-sdk-go v1.42.10
+	github.com/aws/aws-sdk-go v1.44.253
 	github.com/pkg/errors v0.9.1
-	github.com/sirupsen/logrus v1.8.1
+	github.com/sirupsen/logrus v1.9.0
 	github.com/spf13/pflag v1.0.5
-	github.com/stretchr/testify v1.7.0
-	github.com/vmware-tanzu/velero v1.7.1
-	k8s.io/api v0.22.4
-	k8s.io/apimachinery v0.22.4
+	github.com/stretchr/testify v1.8.2
+	github.com/vmware-tanzu/velero v1.10.0-rc.1.0.20230727074327-a6d79fc272a2
```

Comment:
`v1.10.0-rc.1.0.20230727074327-a6d79fc272a2`
The tag and the timestamp are correct, but `v1.10.0` is misleading. 
IMO, `v0.0.0` or `v1.12.0` are better.